collections/esdc-8d-0.083deg-1x2160x4320-2.1.1.yaml

```diff
@@ -0,0 +1,202 @@
+          Description: Net Ecosystem Exchange
+        - Name: net_radiation
+          Description: Net Radiation
+        - Name: open_water_evaporation
+          Description: Open-water Evaporation
+        - Name: ozone
+          Description: Mean Total Ozone Column in dobson units
+        - Name: par
+          Description: Photosynthetically Active Radiation
+        - Name: pardiff
+          Description: Diffuse Photosynthetically Active Radiation
+        - Name: potential_evaporation
+          Description: Potential Evaporation
+        - Name: precipitation
+          Description: Precipitation
+        - Name: precipitation_era5
+          Description: ERA 5 Precipitation
+        - Name: psurf
+          Description: Surface Air Pressure
+        - Name: radiation_era5
+          Description: ERA 5 Surface Shortwave Downwelling Radiation
+        - Name: root_moisture
+          Description: Root-Zone Soil Moisture
+        - Name: sea_ice_fraction
+          Description: sea ice area fraction
+        - Name: sensible_heat
+          Description: Sensible Heat
+        - Name: sm
+          Description: Volumetric Soil Moisture
+        - Name: snow_sublimation
+          Description: Snow Sublimation
+        - Name: snow_water_equivalent
+          Description: Daily Snow Water Equivalent
+        - Name: srex_mask
+          Description: Mask for SREX Regions
+        - Name: stemp
+          Description: Surface Temperature
+        - Name: surface_moisture
+          Description: Surface Soil Moisture
+        - Name: terrestrial_ecosystem_respiration
+          Description: Terrestrial Ecosystem Respiration
+        - Name: totcol_assim
+          Description: Total Ozone Column (Assimilated TM5 data)
+        - Name: totcol_free
+          Description: Total Ozone Column (Assimilated TM5 data)
+        - Name: totcol_msr
+          Description: Total Ozone Column (MSR data)
+        - Name: transpiration
+          Description: Transpiration
+        - Name: water_mask
+          Description: Terrestrial or Water Pixel Classification
+        - Name: water_vapour
+          Description: Total Column Water Vapour
+        - Name: white_sky_albedo
+          Description: White Sky Albedo for Visible Wavebands
+        - Name: white_sky_albedo_avhrr
+          Description: Bi-Hemisphere Reflectance Albedo - VIS band
+        - Name: xch4
+          Description: Column Average Dry-air Mole Fraction Methane
+        - Name: xco2
+          Description: Column Average Dry-air Mole Fraction Carbon Dioxide
+
+Contact: info@earthsystemdatalab.net
+Provider: "[Earth System Data Lab](https://www.earthsystemdatalab.net/)"
+ManagedBy: "[Brockmann Consult](https://www.brockmann-consult.de/)"
+Tags:
+  - Earth System Data Cube
+  - xcube
+  - raster
+  - systematic
+  - open data
+License: |
+  Please refer to individual variables.
+Resources:
+-  Group: xcube Resources
+   DataID: esdc-8d-0.083deg-1x2160x4320-2.1.1.zarr
```

Comment:
If applicable, consider following the structure in [this collection](https://github.com/eurodatacube/public-collections/blob/alicja-adding-ESDC-description/collections/oceancolour-atl-chl-l4-nrt-observations-009-037.yaml) (same as for the other collection).